altair/examples/ridgeline_plot.py

```diff
@@ -0,0 +1,63 @@
+"""
+Ridgeline plot (Joyplot) Example
+--------------------------------
+A `Ridgeline plot <https://serialmentor.com/blog/2017/9/15/goodbye-joyplots>`_
+chart is a chart that lets you visualize distribution of a numeric value for 
+several groups.
+
+Such a chart can be created in Altair by first transforming the data into a
+suitable representation.
+
+"""
+# category: other charts
+import altair as alt
+from vega_datasets import data
+
+source = data.seattle_weather.url
+
+step = 20
+overlap = 1
+
+ridgeline = alt.Chart(source, height=step).transform_timeunit(
```

Comment:
Let's put ``height=step`` into the ``properties()`` call below.

Reply:
done